src/Executables/PrintMinimumGridSpacing/PrintMinimumGridSpacing.cpp

```diff
@@ -35,7 +36,27 @@ void compute_and_print_minimum_grid_spacing(std::string input_file) {
           minimum_grid_spacing(mesh.extents(), map(logical_coordinates(mesh))));
     }
   }
-  printf("The minimum grid spacing is: %1.14e\n", min_grid_spacing);
+  Parallel::printf("The minimum grid spacing is: %1.14e\n", min_grid_spacing);
+}
+
+std::string custom_help_msg() {
+  constexpr int max_label_size = 21;
+  const std::string help_text =
+      "Print the minimum grid spacing between inertial coordinates of the "
+      "Domain specified in the input file. The output can be used to "
+      "choose appropriate time steps.";
+
+  std::ostringstream ss;
+  ss << "\n==== Description of expected options:\n" << help_text
+     << "\n\nOptions:\n"
+     << "  " << std::setw(max_label_size + 2) << std::left << "DomainCreator"
+     << "DomainCreator<Dim, Frame::Inertial>\n"
+     << "  " << std::setw(max_label_size + 2) << std::left << ""
+     << "The domain to create initially.\n"
+     << "  " << std::setw(max_label_size + 2) << std::left << ""
+     << "Dim must correspond to --dimensions arg.\n\n";
+
+  return ss.str();
 }
```

Comment:
This executable takes `--dimensions` as a command line argument to define the value of `Dim` in `OptionTags::DomainCreator<Dim, frame>>`. Assuming a user enters `PrintMinimumGridSpacing --help`, this `custom_help_msg` function is to provide helpful information without the specification of `--dimensions` in lieu of the already-existent `help` function in `src/Options/ParseOptions.hpp`, since I believe the latter would require that `Dim` is already known. 

- See the `help` method here: https://github.com/sxs-collaboration/spectre/blob/a6b583b098b4831aa120d754ad2ce3155670353b/src/Options/ParseOptions.hpp#L497
- See an example usage here: https://github.com/sxs-collaboration/spectre/blob/23053ad76b1188d1e556a374a500f68a5df439c8/src/Parallel/Main.hpp#L185

I think there is probably a cleaner and more robust way of generating this help message, using already-existing functionality. If anyone has comments on a way to improve this, please let me know. The current output of `PrintMnimumGridSpacing --help` looks like this:

```
  -h [ --help ]         Describe program options
  --input-file arg      Input file name
  --dimensions arg      Number of dimensions


==== Description of expected options:
Print the minimum grid spacing between inertial coordinates of the Domain specified in the input file. The output can be used to choose appropriate time steps.

Options:
  DomainCreator          DomainCreator<Dim, Frame::Inertial>
                         The domain to create initially.
                         Dim must correspond to --dimensions arg.
```